src/main/java/io/supertokens/config/CoreConfig.java

```diff
@@ -148,6 +148,9 @@ public class CoreConfig {
     @JsonProperty
     private String supertokens_saas_secret = null;
 
+    @JsonProperty
+    private String supertokens_default_cdi_version = null;
```

Comment:
validate the format of this, and make sure that it is one of the CDIs that is supported in this core version